kata.py

```diff
@@ -0,0 +1,8 @@
+def Add(string):
```

Comment:
Absolute meganitpick: I know the requirements of the kata mentioned `Add` but I think idiomatic Python would have you write `add` for functions (with `Capitalised` names for classes). My assumption is that the kata is writing in Java.